Makefile

```diff
@@ -669,11 +669,21 @@ ssl-cert:
 	openssl req -new -key $(DIR)/tmp/ssl/selfsigned-site.key \
 		-out $(DIR)/tmp/ssl/selfsigned-site.csr -sha256 \
 		-subj '/C=CA/ST=ON/L=Toronto/O=CanDIG/CN=CanDIG Self-Signed Cert'
+	sed -i .tmp s/CANDIG_DOMAIN/$(CANDIG_DOMAIN)/ $(DIR)/etc/ssl/site.cnf
```

Comment:
Instead of `sed` can this CANDIG_DOMAIN be set with the same value as `.env`?

Reply:
Also where does `.tmp` come from?

Reply:
(the .tmp was from a weird issue that I was having with my local version of sed that doesn't seem to happen on dev? So I am taking it back out)

Isn't the sed interpolating the value from .env? I'm not sure what you're suggesting to do instead...?